jekyll/_cci2/github-integration.adoc

```diff
@@ -127,6 +127,16 @@ If your deploy keys or user keys appear to have been removed, it may be due to o
 * When you delete a CircleCI project, CircleCI will remove its associated keys.
 ====
 
+[#create-a-github-user-key]
+=== Create a GitHub user key
+
```

Comment:
Although it's obvious from the heading, I would suggest just adding a short lntro like: "To create a GitHub user key, follow these steps:" or similar